spec/rubocop/cop/rspec/redundant_predicate_matcher_spec.rb

```diff
@@ -0,0 +1,132 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::RedundantPredicateMatcher do
+  it 'registers an offense when using `be_all`' do
+    expect_offense(<<~RUBY)
+      expect(foo).to be_all be_odd
+                     ^^^^^^^^^^^^^ Use `all` instead of `be_all`.
+    RUBY
+
+    expect_correction(<<~RUBY)
+      expect(foo).to all be_odd
+    RUBY
+  end
```

Comment:
Is there an example with `all`?
```ruby
            expect(units).to be_all { |unit| unit.line_item == line_item_2 }
                             ^^^^^^
```
Would the conversion be correct?

Reply:
Since the auto-correction is not correct, would it be better not to make this an offense. So I change not offense for this case.